tests/reverse_diffusion_test.py

```diff
@@ -18,14 +16,7 @@
     # initialize forward diffusion
     fdiff = ForwardDiffusion(beta_start=0.0001, beta_end=0.02, timesteps=256, schedule_type=schedule_type)
 
-    model = Unet(
-        in_channels=2,
-        out_channels=2,
-        dim=32,
-        dim_mults=(1, 2, 4),
-        kernel_size=7,
-        resnet_block_groups=4
-    )
+    model = Unet(in_channels=2, out_channels=2, dim=32, dim_mults=(1, 2, 4), kernel_size=7, resnet_block_groups=4)
```

Comment:
> You should use [magic trailing commas](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) to let black know that you want this split.

It's a suggestion I got from @roszcz, was helpful for those situations. In short, adding a comma at the end of `resnet_block_groups=4` would result in black keeping it not collapsed!